### DISCUSSION TITLE
Should fluidsynth add loop mode release?

### DISCUSSION THREAD

**Discussion by spessasus** [FluidSynth/fluidsynth#1397] [Ideas]
Polyphone 2.4.0 was released. And with it, Davy introduced a looping mode known as "release". It is essentially no loop but it starts when the envelope enters release phase. And in the file, the `sampleModes` flag has a value if **2**.

![image](https://github.com/user-attachments/assets/6dcf9768-3cd7-4538-bdfe-34c678b0b980)

Since people will definitely use this over looping mode 3 with 0-long loops and polyphone is very popular, should fluidsynth add support for it?

**Comment by derselbst** [comment]
Sure, I'm open to add this. It's just, I'm not having the time. I'd welcome a PR for this.
